src/dmd/globals.h

```diff
@@ -233,6 +233,7 @@ struct Param
     // Linker stuff
     Array<const char *> objfiles;
     Array<const char *> linkswitches;
+    Array<bool> linkswitches;
```

Comment:
Duplicated name ? On the D side this is named `linkswitchIsForCC`

Reply:
Heh, I already had this, but apparently wasn't synced correctly on the dmd side.